### PR TITLE
feat: added cards

### PR DESCRIPTION
**Pull request by masraniJay**
Added 7 kinds of cards.
Being late already, I've used the cards I made for assignments.

### CODE REVIEW

**Review by sohan9819 — APPROVED** [review]
looks good to me

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    cursor: pointer;
+    color: #6200ee;
+    background-color: transparent;
+    font-weight: 600;
+    border: none;
+    padding: 10px;
+  }
+
+  .card-button:hover {
+    background-color: #f9f5fe;
```
Comment (ooreally) [inline components/card/card.css:55]: use css variables for colors.

Comment (ooreally) [inline components/card/card.css:55]: use css variables for colors

**Review by ooreally — COMMENTED** [review]
Code looks good. Suggestion:

- use css variables

**Review by Koushith — APPROVED** [review]
LGTM- try to use css variables.

**Pull request merged**